package.json

```diff
@@ -22,8 +22,8 @@
     "test": "node_modules/.bin/mocha"
   },
   "dependencies": {
-    "request": "2.27.*",
     "lodash": "2.2.*",
+    "request": "^2.55.0",
```

Comment:
Unfortunately we have to specify exact major and minor numbers. `"request": "2.55.*"`.